### PR TITLE
Remove dependency on python six from bazel_external_repository_test

### PR DESCRIPTION
**Pull request by aiuto**
The external repository test depends on a .tgz file which would be downloaded from another repo. I think that when whomever made the original test, they just copied six-1.0.0.tar.gz to the test data. But the test does not actually care about what the repo does. It only cares that it can unload it and inject a BUILD file.  This PR replaces that tarball with a trivial equivalent.  

This does not impact the bazel distribution size, it just means we have one less false dependency.

Fixes: #18272

### CODE REVIEW

**Comment by aiuto** [comment]
cc: @meteorcloudy

**Review by meteorcloudy — APPROVED** [review]
Thanks!

**Comment by iancha1992** [comment]
@aiuto can you please add the docstring for `src/test/py/bazel/testdata/bazel_external_repository_test/hello-1.0.0/hello.py`? Thanks!

**Comment by aiuto** [comment]
> @aiuto can you please add the docstring for `src/test/py/bazel/testdata/bazel_external_repository_test/hello-1.0.0/hello.py`? Thanks!

I don't understand what you are asking.  It is a binary source file. They don't have docstrings.

**Comment by Wyverald** [comment]
`hello.py` is not a binary file.

File: src/test/py/bazel/testdata/bazel_external_repository_test/hello-1.0.0/hello.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+def hello():
```
Comment (iancha1992) [inline src/test/py/bazel/testdata/bazel_external_repository_test/hello-1.0.0/hello.py:2]: Need a docstring here @aiuto. Thank you!

**Pull request closed without merging**